src/test/java/HandleTruthTest.java

```diff
@@ -1,8 +1,76 @@
 import org.junit.Test;
 
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+import java.util.TreeMap;
+
+import static junit.framework.TestCase.assertEquals;
+
+@SuppressWarnings("InstantiationOfUtilityClass")
 public class HandleTruthTest {
+    private final HandleTruth WordCount = new HandleTruth ();
+    
+
+
     @Test
     public void makeshiftTest() {
-        HandleTruth.wordCount();
+
+
+       // HandleTruth.phrase();
+    }
+    @Test
+    public void countYour() {
+        Map<? extends Serializable, ?> actualwordCount = new TreeMap<Integer, Set<String>>();
+        final Map<String, Integer> expectedwordCount = new HashMap<>();
+        expectedwordCount.put("your", 1);
+        actualwordCount = HandleTruth.phrase("your");
+        assertEquals(
+                expectedwordCount, actualwordCount
```

Comment:
all the tests are  only verifying a single word? this is essentially writing the same test five times.